crates/cheatcodes/src/fs.rs

```diff
@@ -360,35 +359,37 @@ fn deploy_code(
     value: Option<U256>,
     salt: Option<U256>,
 ) -> Result {
-    let mut bytecode = get_artifact_code(ccx.state, path, false)?.to_vec();
-    if let Some(args) = constructor_args {
-        bytecode.extend_from_slice(args);
-    }
+    // TODO: implement exec_create
 
-    let scheme = if let Some(salt) = salt {
-        revm::primitives::CreateScheme::Create2 { salt }
-    } else {
-        revm::primitives::CreateScheme::Create
-    };
+    // let mut bytecode = get_artifact_code(ccx.state, path, false)?.to_vec();
+    // if let Some(args) = constructor_args {
+    //     bytecode.extend_from_slice(args);
+    // }
 
-    let outcome = executor.exec_create(
-        CreateInputs {
-            caller: ccx.caller,
-            scheme,
-            value: value.unwrap_or(U256::ZERO),
-            init_code: bytecode.into(),
-            gas_limit: ccx.gas_limit,
-        },
-        ccx,
-    )?;
+    // let scheme =
+    //     if let Some(salt) = salt { CreateScheme::Create2 { salt } } else { CreateScheme::Create
+    // };
 
-    if !outcome.result.result.is_ok() {
-        return Err(crate::Error::from(outcome.result.output))
-    }
+    // let outcome = executor.exec_create(
+    //     CreateInputs {
+    //         caller: ccx.caller,
+    //         scheme,
+    //         value: value.unwrap_or(U256::ZERO),
+    //         init_code: bytecode.into(),
+    //         gas_limit: ccx.gas_limit,
+    //     },
+    //     ccx,
+    // )?;
 
-    let address = outcome.address.ok_or_else(|| fmt_err!("contract creation failed"))?;
+    // if !outcome.result.result.is_ok() {
+    //     return Err(crate::Error::from(outcome.result.output))
+    // }
 
-    Ok(address.abi_encode())
+    // let address = outcome.address.ok_or_else(|| fmt_err!("contract creation failed"))?;
+
+    // Ok(address.abi_encode())
+
+    Ok(Default::default())
```

Comment:
To do: restore `deploy_code` cheatcode implementation

I think this is the only place that uses `exec_create` (which in turn uses `with_evm`), let's see if we can simplify and inline